crates/bevy_ecs/src/event/base.rs

```diff
@@ -16,7 +17,7 @@ use std::{
     label = "invalid `Event`",
     note = "consider annotating `{Self}` with `#[derive(Event)]`"
 )]
-pub trait Event: Send + Sync + 'static {}
+pub trait Event: Component {}
```

Comment:
These docs really need some breadcrumbs explaining what it means for an event to be used as a component.